build.gradle

```diff
@@ -83,11 +84,30 @@ if (hasProperty('buildScan')) {
     }
 }
 
-tasks.withType(FindBugs) {
+spotbugs {
+    effort = 'max'
+    // This makes the "html" reports come out in plain text so you can just open the file in IntelliJ
+    // and look at your bugs instead of having to switch to a browser.
+    extraArgs = [ '-emacs']
+}
+spotbugsMain {
     reports {
-        xml.enabled false
-        html.enabled true
-        html.stylesheet resources.text.fromFile('config/findbugs/plain.xsl')
+        html {
+            enabled = true
+            destination = file("$buildDir/reports/spotbugs/main.txt")
+            stylesheet = "plain.xsl"
```

Comment:
Lol, you don't love their sweet html reports?

Reply:
I think I've switched it back to just the default html reports